### PR TITLE
ariang: add new package

### PR DESCRIPTION
**Pull request by Ansuel**
This is a porting to include another webui interface for aria2. I tested it and works good.

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

**Comment by diizzyy** [comment]
Is there a reason why you can't use the release?
https://github.com/mayswind/AriaNg/releases

**Comment by Ansuel** [comment]
@diizzyy New feature that are not present in release. I can switch to 0.2.0 if necessary

**Comment by Ansuel** [comment]
Hello ?

**Comment by diizzyy** [comment]
Please stick to releases unless it's really the end of the world if its it's not in ;-)

**Comment by Ansuel** [comment]
@diizzyy should be good now

**Comment by Ansuel** [comment]
Now i remember why i not used release.... 
I didn't like the use of PKG_UNPACK ahahaha 
looks terrible to me

**Comment by Ansuel** [comment]
@diizzyy Hello ?

**Comment by diizzyy** [comment]
Hmm? You can use https://github.com/mayswind/AriaNg/archive/0.2.0/AriaNg-0.2.0.tar.gz as URL? :-)

**Comment by Ansuel** [comment]
@diizzyy nope... that is code to be compiled... that's the problem

**Comment by Ansuel** [comment]
@diizzyy Can we include this?

**Comment by diizzyy** [comment]
Sure, LGTM

**Comment by Ansuel** [comment]
@diizzyy why not included?

**Comment by Ansuel** [comment]
@diizzyy any update?

**Comment by Ansuel** [comment]
add this pls... why wait

**Comment by diizzyy** [comment]
What does this bring to the table compared to webui-aria2?

**Comment by Ansuel** [comment]
@diizzyy better and different ui... 
(we have 2 webui so why not add this too )

**Comment by Ansuel** [comment]
@diizzyy any problem?

**Comment by diizzyy** [comment]
My personal opinion is that we don't need three different packages which essentially overlaps each others functionality. In an ideal world you'd have a dialog with @kuoruan and agree on providing one webui.

File: net/ariang/Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+define Package/ariang/install
+	$(INSTALL_DIR) \
+		$(1)/www/ariang
+
+	$(CP) \
+		$(PKG_BUILD_DIR)/{langs,css,js,fonts} \
+		$(PKG_BUILD_DIR)/index.html \
+		$(PKG_BUILD_DIR)/index.manifest \
+		$(PKG_BUILD_DIR)/LICENSE \
+		$(1)/www/ariang
```
Comment (kuoruan) [inline net/ariang/Makefile]: Copy ```robots.txt``` and delete ```index.manifest```?

Comment (Ansuel) [inline net/ariang/Makefile]: done

**Comment by Ansuel** [comment]
@diizzyy why? different gui gives different type of info... (some are lighter than others for example)

**Comment by kuoruan** [comment]
I well consider remove yaaw or webui-aria2. Any opinion?

**Comment by Ansuel** [comment]
@kuoruan webui-aria2 is good (active development and translated to many language). ariang is the one with more option and info than others but i didn't check the actual size of the webui...
yaaw (well... development stop 1 year ago so...) 
For me keep this 2... What do you think ?

**Comment by leonghui** [comment]
Just my 2 cents, I've used both yaaw and ariang. I suggest adding ariang as an option.

yaaw
- pro: small unpacked size (413 kB), simple & fast UI
- con: fails to load on Android Webview-based browsers (e.g. Lightning, Firefox Focus), use case is adding downloads and checking on status remotely
- con: author merged minor PRs but doesn't seem to be keen on updating it otherwise: https://github.com/binux/yaaw/issues/79#issuecomment-148848165

ariang
- con: much larger (2.29 MB), consider removing font files (928 kB)
- pro: desktop and mobile interfaces, works with various Android browsers
- pro: beautiful and functional UI (realtime speed graph), more options when starting new downloads, full implementation of changeGlobalOption method (update Aria2 config without restarting service)

**Comment by Ansuel** [comment]
@kuoruan  also ariang show if you don't have disk space, (webui-aria2 and yaaw not...)

**Comment by kuoruan** [comment]
@Ansuel This url may be better https://github.com/mayswind/AriaNg-DailyBuild

**Comment by Ansuel** [comment]
@kuoruan dizzy told me that we should use release... 
In the first "version" of this PR i used that link... 
Don't know what to do LOL 
Use release or use daily?

**Comment by Ansuel** [comment]
@diizzyy can this be pushed? can't underesttand the problem in having multiple webui for aria2....

**Comment by diizzyy** [comment]
@Ansuel 
No longer involved with OpenWrt so you need to ask committers what they think.

**Comment by Ansuel** [comment]
@hnyman any chance?

**Pull request closed without merging**